benchmarks/Bio/BCR.jmd

```diff
@@ -90,24 +97,62 @@ test_sol  = TestSolution(sol)
 
 ## Setups
 
+#### Declare pre-conditioners
+```julia
+using IncompleteLU, LinearAlgebra
+
+jaccache = sparsejacprob.f.jac(oprob.u0,oprob.p,0.0)
+W = I - 1.0*jaccache
+prectmp = ilu(W, τ = 50.0)
+preccache = Ref(prectmp)
+
+τ1 = 1e2

```

Comment:
These two tau choices are not the same?

Reply:
So, for the CVODE pre-conditioner, in https://benchmarks.sciml.ai/dev/StiffODE/Bruss/ there were two tau values, I tried changing both, but only one actually seemed to affect the runtime (the other one I left at its value of 50).

For the tau between the CVODE and Julia pre-conditioners,  I just tried various values, and the tau's that optimised the runtimes were different between the two.